app/policies/topic_policy.rb

```diff
@@ -0,0 +1,2 @@
+class TopicPolicy < ApplicationPolicy
+end
```

Comment:
looks like an empty line missing here and on `app/views/api/v1/topics/index.json.jbuilder`, its a good practice to leave an empty line on the end of the file